app/views/admin/ndbn_members/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<section class="content-header">
+  <div class="container-fluid">
+    <div class="row mb-2">
+      <div class="col-sm-6">
+        <% content_for :title, "Community - NDBN Member Upload" %>
+        <h1>NDBN Member Upload</h1>
+      </div>
+      <div class="col-sm-6">
+        <ol class="breadcrumb float-sm-right">
+          <li class="breadcrumb-item"><%= link_to(admin_ndbn_members_path) do %>
+              <i class="fa fa-dashboard"></i>
+              Home
+            <% end %>
+          </li>
+        </ol>
+      </div>
+    </div>
+  </div><!-- /.container-fluid -->
+</section>
+
+<section class="content">
+  <div class="container-fluid">
+    <div class="row">
+      <!-- left column -->
+      <div class="col-md-12">
+        <!-- jquery validation -->
+        <div class="card card-primary">
+          <div class="card-header">
+            <h3 class="card-title">Upload NDBN Members</h3>
+          </div>
+          <!-- /.card-header --> <!-- form start -->
+          <div class="card-body">
+            <div class="row">
+              <p>Update the current NDBN members by uploading a CSV file of the
+                current members.</p>
```

Comment:
Can we add some instructions here as to what format we expect?

Reply:
@cielf any input on what the instructions should be? That fine?

![image](https://github.com/rubyforgood/human-essentials/assets/14540596/e868e7c1-8ca4-4536-bd1b-e07a75d647ff)

Reply:
We could give explicit instructions to say "The Current NDBN member list is found at https://docs.google.com/spreadsheets/d/1DRXeG206M0kgtUYT9lpD8fyEWRO7-a6m/edit?pli=1#gid=1068714915 .
Download the sheet, and remove the initial 'Updated' row.

If they change it again,  we can change the instructions as a Good First Issue.

Reply:
![image](https://github.com/rubyforgood/human-essentials/assets/14540596/3c814dc3-bdcd-4630-977a-5f4d7fff5939)

New look